sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -27,7 +27,7 @@ export enum AppRoutes {
   ManageInstitutionDesignation = "manage-desgination",
   InstitutionManageUsers = "manage-users",
   DisabledUser = "login/disabled-user",
-  StudentApplication = "application-form/:id?/:selectedForm/:programYearId",
+  StudentApplication = "application-form/programYear/:selectedForm/:programYearId/application/:id",
```

Comment:
Can we have something like this for the route? "application-form/:selectedForm/programYear/:programYearId/application/:id"

Reply:
Yes, it makes more sense 👍

Reply:
Done